python/paddle/distributed/auto_parallel/api.py

```diff
@@ -826,9 +827,23 @@ def replicate_layer_params_and_buffers(
         )
 
 
+def get_value_placement(value):
+    dist_attr = value.dist_attr()
+    assert dist_attr is not None, "Can't get placement for a dense value."
+    mesh = dist_attr.process_mesh
+    dims_mapping = dist_attr.dims_mapping
+    partial_status = dist_attr.partial_status
+    palcements = to_placements(dims_mapping, mesh, partial_status)
+    return palcements
```

Comment:
We can add `placement` as  Value's property in the future to make it dynamic-static consistent